### ISSUE TITLE
no https specifiers that are not using deno.land/x

### ISSUE THREAD

**Issue by ry** [denoland/deno-docs#443]
For example, here is a code snippet that uses `x.nest.land`
https://docs.deno.com/runtime/manual/basics/modules/#remote-import

```
import {
  add,
  multiply,
} from "https://x.nest.land/ramda@0.27.0/source/index.js";

function totalCost(outbound: number, inbound: number, tax: number): number {
  return multiply(add(outbound, inbound), tax);
}

console.log(totalCost(19, 31, 1.2));
console.log(totalCost(45, 27, 1.15));
```

switch all https imports to https://deno.land/x imports so we can ensure they continue to work

(also that code sample has an annoying amount of comments in it - those should be remove too)

**Comment by thisisjofrank** [comment]
there are lots of imports that are currently on esm.sh which as far as I can see are modules that we don't have on deno.land/x
are these ok to leave as are?

**Comment by thisisjofrank** [comment]
https://github.com/denoland/deno-docs/pull/529

**Comment by ry** [comment]
closed in #529
